packages/did-core-test-server/package.json

```diff
@@ -20,10 +20,18 @@
     "fastify-cors": "^5.0.0",
     "fastify-swagger": "^3.5.0",
     "jest": "^26.6.3",
+    "jest-did-matcher": "file:../jest-did-matcher",
```

Comment:
it is really not possible to install this as a dependency?

You can also use allow `lerna boostrap` to auto link dependencies like this.

Reply:
this also raises questions about npm and publishing... maybe we should publish this package?

Reply:
Ok. I will learn about `lerna` a bit more.

Whether we should publish or not -- publishing this matcher package, or the entire suite?
Publishing jest-did-matcher separately might be useful in certain contexts.